src/node_platform.cc

```diff
@@ -13,9 +15,9 @@ using v8::Platform;
 using v8::Task;
 using v8::TracingController;
 
-static void FlushTasks(uv_async_t* handle) {
-  NodePlatform* platform = static_cast<NodePlatform*>(handle->data);
-  platform->FlushForegroundTasksInternal();
+void PerIsolatePlatformData::FlushTasks(uv_async_t* handle) {
```

Comment:
Is it possible to move this function to where all the other `PerIsolatePlatformData` methods are?